forge-gui/res/cardsfolder/upcoming/stirge.txt

```diff
@@ -0,0 +1,8 @@
+Name:Stirge
+ManaCost:B
+Types:Creature Insect Bat
+PT:1/1
+A:AB$ Draw | Cost$1 B PayLife<1> Sac<1/CARDNAME> | NumCards$ 1 | SpellDescription$ Draw a card.
```

Comment:
add space – `Cost$ 1`

See how `Defined$ You` is the default here so you didn't need it?
Last one, I promise: `NumCards$ 1` is also default for Draw Effect